test/integration/cart_integration_test.rb

```diff
@@ -222,12 +166,11 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
     assert_equal "/pending_loan", current_path
     within("#pending_loans") do
       assert page.has_content?(tenant.projects.first.title)
-      assert page.has_content?("13000")
+      assert page.has_content?(tenant.projects.first.price)
     end
   end
 
-  test "authed user can select a diff amt for a loan add it to their cart" do
-    skip
+  test "an authed user can select a diff amt for a loan add it to their cart" do
```

Comment:
I think this test should probably be renamed since the user isn't actually selecting a different amount.